src/languages/types.ts

```diff
@@ -122,6 +122,8 @@ type PayerSettledParams = {amount: number};
 
 type WaitingOnBankAccountParams = {submitterDisplayName: string};
 
+type CanceledRequestParams = {amount: string; submitterDisplayName: string};
+
 type SettledAfterAddedBankAccountParams = {submitterDisplayName: string; amount: string};
```

Comment:
These two types are the same, can we merged them?

Reply:
@robertKozik https://github.com/Expensify/App/blob/ba75ce9734c87062199e57d16f706724c2374447/src/languages/types.ts#L103

https://github.com/Expensify/App/blob/ba75ce9734c87062199e57d16f706724c2374447/src/languages/types.ts#L109

https://github.com/Expensify/App/blob/ba75ce9734c87062199e57d16f706724c2374447/src/languages/types.ts#L107

there are a lot of duplicate types, don't you think its better to keep it for its clear naming? if we want to add another param it'll be easier since we have types for each function,  if not what should we name the type after combining it as one.